### PR TITLE
building the first page

### PR DESCRIPTION
**Pull request by shamskhodary**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,75 @@
...
+            <h4 class="subtitle">Everyone can be a chef</h4>
+          </div>
+          <div class="num-btn">
+            <label for="phone"></label>
+            <input
+              type="tel"
+              id="phone number"
+              placeholder="Phone Number"
+              class="input"
+            />
```
Comment (mo7mad4) [inline index.html:36]: adding number Such as +62 and down arrow

**Review by mo7mad4 — DISMISSED** [review]
Good job Shams

**Review by MarwaKHTanani — APPROVED** [review]

**Review by azzsal — CHANGES_REQUESTED** [review]
Your branch has some conflicts with the main branch.

**Review by azzsal — APPROVED** [review]

File: style.css [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+}
+
+.box,
+.box2,
+.box3 {
+  display: flex;
+  background: #fff;
+  border: 1px solid #b3b3b3;
+  border-radius: 12px;
+  height: 56px;
```
Comment (MarwaKHTanani) [inline style.css:146]: I guess you should add "justify-content: center;" here

File: style.css [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+}
+
+.g-title {
+  position: relative;
+  right: -9px;
+}
+/* Condition section  */
+
+.condition {
+  margin: 90px 0px;
```
Comment (MarwaKHTanani) [inline style.css:190]: changing this to margin: 40px 0px; making it better

File: style.css [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+  text-align: center;
+}
+
+p {
+  font-family: Cera pro;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 20px;
+  font-size: 16px;
```
Comment (MarwaKHTanani) [inline style.css:199]: edit this to font-size: 12px;

**Review by MarwaKHTanani — CHANGES_REQUESTED** [review]
edit

**Review by ShbairAli — APPROVED** [review]

**Review by MarwaKHTanani — APPROVED** [review]

**Review by ShbairAli — APPROVED** [review]

**Review by mo7mad4 — APPROVED** [review]

**Review by ShbairAli — APPROVED** [review]

**Review by MarwaKHTanani — APPROVED** [review]

**Review by azzsal — APPROVED** [review]

**Pull request merged**